src/Microsoft.PowerShell.ConsoleHost/WindowsTaskbarJumpList/ComInterfaces.cs

```diff
@@ -10,30 +10,30 @@ namespace Microsoft.PowerShell
 {
     internal static class ComInterfaces
     {
-        [DllImport("kernel32.dll", SetLastError = true, EntryPoint = "GetStartupInfoA")]
+        [DllImport("kernel32.dll", SetLastError = false, EntryPoint = "GetStartupInfoW")]
         internal static extern void GetStartupInfo(out StartUpInfo lpStartupInfo);
 
-        [StructLayout(LayoutKind.Sequential, CharSet = CharSet.Unicode)]
+        [StructLayout(LayoutKind.Sequential)]
```

Comment:
Please add a comment why we use IntPtr for string - this help us avoid problems in future.